app/models/snapshot/adviser_queries.rb

```diff
@@ -0,0 +1,132 @@
+module Snapshot::AdviserQueries
+  def query_registered_advisers
+    Adviser.all
+  end
+
+  def query_advisers_in_england
+    advisers_in_country(Adviser.all, 'England')
+  end
+
+  def query_advisers_in_scotland
+    advisers_in_country(Adviser.all, 'Scotland')
+  end
+
+  def query_advisers_in_wales
+    advisers_in_country(Adviser.all, 'Wales')
+  end
+
+  def query_advisers_in_northern_ireland
+    advisers_in_country(Adviser.all, 'Northern Ireland')
+  end
+
+  TravelDistance.all.keys.each do |val|
+    method_name = val.downcase.gsub(' ', '_')
+    define_method "query_advisers_who_travel_#{method_name}" do
+      advisers_who_travel(val)
+    end
+  end
```

Comment:
What would happen if we were asked to change the TravelDistances? Would the snapshots cope or would they need to be updated?